doc/source/Resources/toolkit_installer_from_aedt.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+import sys
+
+# This script installs PyAEDT tabs (PyAEDT Console, Jupyter, Run Script and Extension Manager)
+# using a specific Python interpreter.
+# It can be passed from an environment variable called "PYAEDT_INTERPRETER" or from the input argument.
+# The environment variable has more priority.
+
+is_linux = os.name == "posix"
+is_windows = not is_linux
+
+pyaedt_enviroment_variable = "PYAEDT_INTERPRETER"
+
+
+def run_pyinstaller_from_c_python(oDesktop, pyaedt_interpreter):
+    # Iron Python script to create PyAEDT panels
+
+    # Get AEDT information
+    version = oDesktop.GetVersion()[2:6].replace(".", "")
+
+    # Add PyAEDT tabs in AEDT
+
+    # Create Toolkits in PersonalLib
+    import tempfile
+    python_script = os.path.join(tempfile.gettempdir(), "configure_pyaedt.py")
+    if os.path.isfile(python_script):
+        os.remove(python_script)
+    with open(python_script, "w") as f:
+        f.write("from ansys.aedt.core.workflows.installer.pyaedt_installer import add_pyaedt_to_aedt\n")
+        f.write(
+            'add_pyaedt_to_aedt(aedt_version="{}", personal_lib=r"{}")\n'.format(
+                oDesktop.GetVersion()[:6], oDesktop.GetPersonalLibDirectory()))
+
+    command = r'"{}" "{}"'.format(pyaedt_interpreter, python_script)
+    oDesktop.AddMessage("", "", 0, "Configuring PyAEDT panels in automation tab.")
+    ret_code = os.system(command)
```

Comment:
```suggestion
    command = [f'"{pyaedt_interpreter}"', '"{python_script}"')
    oDesktop.AddMessage("", "", 0, "Configuring PyAEDT panels in automation tab.")
    ret_code = subprocess.run(command)
```
Here I'm changing the previous behavior to avoid shell injection.